nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/ui.js

```diff
@@ -3,4 +3,10 @@
 module.exports = function () {
   this.When('I click the "$button" button', (button) => this.ui.drawer.open(button));
   this.When('I select "$option" from the View menu', (option) => this.ui.view(option));
+  this.When('I reload the page', () => this.ui.reload());
+  this.Then('I can see "$activity" in the Activity feed', (activity) => {
+    this.ui.activityFeed.waitForVisible();
+    this.ui.getActivity(activity).isVisible().should.be.true;
```

Comment:
It would be more consistent to have`this.ui.activityFeed.getActivity(activity)` instead.